plotly/tools.py

```diff
@@ -6208,6 +6207,9 @@ def create_dendrogram(X, orientation="bottom", labels=None,
         if len(s) != 2:
             exceptions.PlotlyError("X should be 2-dimensional array.")
 
+        if distfun is None:
+            distfun = scs.distance.pdist
+
```

Comment:
Got it. should `scs` by `scp` in this case? Also, what happens if a user runs this function but they don't have `scipy` installed? It's not a required dependency of the `plotly` package (and it shouldn't be) but folks might be surprised if running the function fails due to a missing module. 

What do we do in other parts of the code that use `scipy`? Do we just let it fail or do we tell users that they need to install `scipy`?